diff --git a/src/modules/ipc.cpp b/src/modules/ipc.cpp
index 18a6c3e1..f8d077c5 100644
--- a/src/modules/ipc.cpp
+++ b/src/modules/ipc.cpp
@@ -62,7 +62,8 @@ namespace modules {
     if (m_initial) {
       auto command = command_util::make_command(m_hooks.at(m_initial - 1)->command);
       command->exec(false);
-      command->tail([this](string line) { m_output = line; });
+      m_output = command->readline();
+      //command->tail([this](string line) { m_output = line; });
     }
     static_module::start();
   }
